.github/workflows/ci.yml

```diff
@@ -25,7 +25,7 @@ jobs:
       id-token: write
       contents: read
 
-    environment: dev
+    environment: development
```

Comment:
this will need to be `dev` as it refers to the GitHub environments (i.e. where to retrieve secrets)